plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -337,6 +337,12 @@ bool GazeboYarpControlBoardDriver::gazebo_init()
                 m_coupling_handler.push_back(cpl);
                 yCInfo(GAZEBOCONTROLBOARD) << "using icub_left_hand_mk4";
             }
+            else if (coupling_bottle->get(0).asString()=="hand_mk5")
```

Comment:
Can we add an `icub_` prefix here? It is not ideal to continue to put model-specific information here (see https://github.com/robotology/gazebo-yarp-plugins/issues/470), but it we do it let's at least make it clear in the public parameters to what robot/hands "family" we are referring to. Furthemore, for the `hand_mk4` the coupling were left specific, see https://github.com/robotology/gazebo-yarp-plugins/pull/620#issuecomment-1087364167 . Do you have any idea why instead for the mk5 this is not a problem?

Reply:
I changed the name in `icub_hand_mk5` and as far as I know, the mk4 coupling contained the `left` in the name because there was only a left hand for mk4, but trying it seems that that coupling is ok also for the right.

Reply:
For reference: we checked with @mfussi66 and he also confirmed that the coupling is not left or right specific, this was his assumption in https://github.com/robotology/gazebo-yarp-plugins/pull/620#issuecomment-1087364167 but it turned out that it was not true.